index.js

```diff
@@ -1,7 +1,9 @@
 #! /usr/bin/env node
 const shell = require('shelljs')
-const { path } = require('app-root-path')
-const fs = require("fs")
+const { path: appRoot } = require('app-root-path')
+const fs = require('fs')
+const os = require('os')
+const join = require('path').join
```

Comment:
Why not destructure like above?

Reply:
Saw that tool is not using a bundler - yet no tree-shaking - and that wouldn't impact.

but since you have commented, then you got it :') I will commit a destructed one for you